### PR TITLE
feat: input css

### PR DESCRIPTION
**Pull request by Dart-EY**
input.css file to address the design changes in the Semantic UI library for the `Input` component.

### CODE REVIEW

File: src/components/pages/Theme/Theme.jsx [hunk 1]
```diff
@@ -15,6 +15,7 @@ import {
 
 // styles
 import './Theme.css';
+import '../../../styles/input.css';
```
Comment (michael-lynch) [inline src/components/pages/Theme/Theme.jsx]: We need to include this in `ga.css`. That's the only file FAST is going to be referencing.

File: src/ga.css [hunk 2]
```diff
@@ -5,5 +5,6 @@
 @import 'styles/dropdown.css';
 @import 'styles/form.css';
 @import 'styles/header.css';
+@import 'styles/form.css';
```
Comment (michael-lynch) [inline src/ga.css]: We already have `form.css` but we'll need `input.css` here.

**Review by michael-lynch — CHANGES_REQUESTED** [review]

File: src/components/pages/Theme/Theme.jsx (hunk 1 above)
Comment (alva-fresh) [reply src/components/pages/Theme/Theme.jsx:18]: Can we merge all style files for components to `ga.css` just before we hand it off to FAST? And for now keep them separate, so it's easier to review

File: src/styles/input.css [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+.ui.input.error > input:focus::placeholder {
+  color: var(--neutral-900);
+  font-weight: 400;
+}
+
+.ui.input.disabled > input {
+  border: 1px solid var(--neutral-700);
+}
+
+.ui.input.disabled > inpu::placeholder {
```
Comment (alva-fresh) [inline src/styles/input.css]: `input::placeholder` here

File: src/styles/input.css [hunk 4]
```diff
@@ -0,0 +1,41 @@
+.ui.input > input {
+  padding: var(--spacing-1) var(--spacing-4) var(--spacing-1) var(--spacing-2);
+  line-height: 24px;
+  border-radius: var(--border-radius-2);
+  background-color: var(--shade-white);
+  border: 1px solid var(--neutral-900);
+  font-size: 16px;
```
Comment (alva-fresh) [inline src/styles/input.css]: Can we make variables and then reference them?
```
--font-size: 16px;
--line-height: 24px;
```

File: src/styles/input.css [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+}
+
+.ui.input > input:focus {
+  border-color: var(--primary-pms-362_900);
+}
+
+.ui.input > input::placeholder {
+  color: var(--neutral-900);
+  font-weight: 400;
```
Comment (alva-fresh) [inline src/styles/input.css:16]: `--book-weight`

File: src/ga.css (hunk 2 above)
Comment (Dart-EY) [reply src/ga.css]: I might have lost it in the rebase. sorry! Just updating it!

File: src/styles/input.css (hunk 4 above)
Comment (Dart-EY) [reply src/styles/input.css]: Definitely!

File: src/components/pages/Theme/Theme.jsx [hunk 6]
```diff
@@ -15,6 +15,7 @@ import {
 
 // styles
 import './Theme.css';
+import '../../../ga.css';
```
Comment (alva-fresh) [inline src/components/pages/Theme/Theme.jsx]: I think it's already in `index.js`, so you don't need to import it here.
```
// styles
import 'semantic-ui-css/semantic.css';
import './ga.css';
```

File: src/styles/variables.css [hunk 7]
```diff
@@ -55,6 +55,8 @@
 
   /* font */
   --gotham: 'Gotham', Arial, Calibri, sans-serif;
+  --font-size: 16px;
+  --line-height: 24px;
```
Comment (michael-lynch) [inline src/styles/variables.css]: Let's use `rem` values for these.

File: src/styles/variables.css [hunk 8]
```diff
@@ -92,4 +94,9 @@
   /* transition */
   --transition-hover: 200ms;
   --transition-appear: 500ms;
+
+  /* weights */
+  --black-weight: 900;
+  --bold-weight: 700;
+  --book-weight: 400;
```
Comment (michael-lynch) [inline src/styles/variables.css]: These are basically 1-1 mappings, so I don't think we need them. This was more relevant for Semantic themes, which referenced variables outside the scope of a particular theme. But in our case, we can just reference `900`, `700` etc. directly.

Comment (alva-fresh) [reply src/styles/variables.css]: We used to have them in our Storybook library, though

Comment (Dart-EY) [reply src/styles/variables.css]: Should I remove them, then? I see value in keeping it if future changes are necessary, so we only need to correct it in one place.

Comment (michael-lynch) [reply src/styles/variables.css]: That's a good point @Dart-EY but I wouldn't want to change it to something like `--book-weight: 900`. That will get really confusing. 

Personally I don't think we need them, but if you want to keep them, we can, but I think in that case, they need to be contextual. Something like...

```
--weight-body: 400;
--weight-heading: 700;
--weight-display: 900;
```

File: src/components/pages/Theme/Theme.jsx (hunk 6 above)
Comment (Dart-EY) [reply src/components/pages/Theme/Theme.jsx:18]: It didn't work, I am taking a look on why was that

File: src/styles/variables.css (hunk 8 above)
Comment (Dart-EY) [reply src/styles/variables.css]: Got it! I will keep them then, but change the naming to the above!

File: src/App.css [hunk 9]
```diff
@@ -21,3 +21,7 @@ body {
   padding: var(--spacing-2);
   font-family: var(--gotham);
 }
+
+html {
+  font-size: 16px;
```
Comment (alva-fresh) [inline src/App.css:26]: We might want to let Fast know that we're using `rem` values for sizes. So, they need to make sure it corresponds to `16px` on their end.

File: src/styles/variables.css [hunk 10]
```diff
@@ -92,4 +94,9 @@
   /* transition */
   --transition-hover: 200ms;
   --transition-appear: 500ms;
+
+  /* weights */
+  --weight-body: 400;
```
Comment (alva-fresh) [inline src/styles/variables.css]: What do you think of this comment?
https://github.com/abby-moffitt/GA-Semantic/pull/3#discussion_r1198339778

Comment (Dart-EY) [reply src/styles/variables.css]: Yeah! I see where the problem is now :/ I will update this PR accordingly

**Review by alva-fresh — APPROVED** [review]

**Pull request merged**